src/components/atoms/FormattedInput/FormattedInput.stories.tsx

```diff
@@ -121,3 +121,20 @@ export const PresetValue: Story = {
   },
   render: (args) => <PresetValueTemplate {...args} />,
 };
+
+export const Variants: Story = {
```

Comment:
if the functionality of this component is not affected by `variant` we should not add these stories